daprdocs/content/en/getting-started/quickstarts/cryptography-quickstart.md

```diff
@@ -0,0 +1,113 @@
+---
+type: docs
+title: "Quickstart: Cryptography"
+linkTitle: Cryptography
+weight: 79
+description: Get started with the Dapr Cryptography building block
+---
+
+{{% alert title="Note" color="primary" %}}
+The cryptography building block is currently in **alpha**. 
+{{% /alert %}}
+
+Let's take a look at the Dapr [cryptography building block](todo). In this Quickstart, you'll create an application that encrypts and decrypts data using the Dapr cryptography APIs (high-level). You'll:
```

Comment:
```suggestion
Let's take a look at the Dapr [cryptography building block](todo). In this Quickstart, you'll create an application that encrypts and decrypts data using the Dapr cryptography APIs. You'll:
```

I would like to stay away from the using the term "high level" since this is confusing and subjective. They are all APIs on a building block. Some (in the future) may be harder to use than others, but that is when/if we decide to add them.